LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 SourceBots
```

Comment:
This isn't really how copyright works... it's more usual to include the individual authors, unless you're asking for an assignment of copyright here?

Reply:
If we _were_ going to keep the copyright ownership as "SourceBots", it would also be worth defining what entity looks like. Currently it's just an independant group of people, including but not limited to those in the org. Not sure that can own any copyright per say.

Reply:
`robot-api` is licenced to "SourceBots": https://github.com/sourcebots/robot-api/blob/master/LICENSE
We should probably be consistent.

Reply:
> Copyright (c) 2017 Source Bots

No idea who "Source Bots" are. We're "SourceBots".

Reply:
> No idea who "Source Bots" are. We're "SourceBots".

It would be useful if this was taken seriously.

Reply:
Ok, so if you're asking for an assignment of copyright then I think we do need there to be an actual (probably legally registered) entity which can own the copyright. If someone has investigated and discovered that we don't need the owning entity to be legally registered I'm happy to change my opinion here.

Let's not worry about the other repo for now -- we can come back and fix it with whatever we decide the right course is, I think we're in agreement that `robot-api` isn't really licensed well at the moment.

So, alternatives then.

I think it would be reasonable to license as being authored by "SourceBots Contributors" perhaps with a comment to look in the git history to see exactly who that means, or possibly to have a separate AUTHORS file which is kept up to date (CI can help on that).

Alternatively I have seen cases where the MIT license is specified without author names, again implying that the reader should look in the source history.

Finally, there's listing all the names of the contributors. Again this would ideally be kept in step  by CI, but that might be overkill. For reference, here's the list of contributors grouped by contribution ranges:
```
2017 Alistair Lynn, Thomas Leese
2017-2018 Jake Howard, Kier Davis, Peter Law
2018 Dan Trickey
```
Since there haven't been any changes this year (and I don't think that adding a license file qualifies as a copyrightable change), there's no mention of 2019 here.

Here's what that might look like in the file:
```suggestion
Copyright (c) 2017 Alistair Lynn, Thomas Leese
              2017-2018 Jake Howard, Kier Davis, Peter Law
              2018 Dan Trickey
```

I'm fairly happy with any of these solutions, with a preference for the latter. As noted I don't think that the copyright being attributed directly to SourceBots is workable.

Reply:
Copyright is able to be assigned to organisations that are not necessarily legal entities, for example an unincorporated association. This is similar to how an unincorporated association can possess assets, bank accounts, etc. Unfortunately "SourceBots" is not an unincorporated association, and to the best of my knowledge, we cannot actually assign a copyright to it. I apologise for my assumption that we could based off other repositories.

I think this partially boils down to "SourceBots" being somewhat loosely defined. If we were to make it an unincorporated association, it would need a constitution etc. This would also raise the question of whom SourceBots' assets belong to, which to my understanding is a mix of the University of Southampton and Southampton Robotics Outreach. I appear to have digressed and suggest that this issue is discussed at another time. I have opened an issue about the copyright assignment on `robot-api`.

For now, I think that Peter's suggestion is probably the easiest, so I will commit that to this PR.